src/services/map-events-mananger/event-registration-input.ts

```diff
@@ -8,7 +8,7 @@ import { PickOptions } from './consts/pickOptions.enum';
  * MapEventsManagerService.register(eventRegistrationInput).subscribe()
  * ```
  */
-export interface EventRegistrationInput {
+export abstract class EventRegistrationInput {
```

Comment:
We had a problem with an empty file.
Tzahi and I, checked Angular2 package standards and they actually change interfaces into abstract classes - so it compiles to an empty object